modules/databend/examples_test.go

```diff
@@ -0,0 +1,87 @@
+package databend_test
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/databend"
+)
+
+func TestExampleRun(t *testing.T) {
```

Comment:
We do not add testing code in the testable examples file, as they will land in the pkg.go.dev site as code snippets. Please see https://go.dev/blog/examples as reference

Reply:
Got it.